src/main/webapp/app/entities/modeling-exercise/modeling-exercise.service.ts

```diff
@@ -19,59 +19,33 @@ export class ModelingExerciseService {
     constructor(private http: HttpClient) {}
 
     create(modelingExercise: ModelingExercise): Observable<EntityResponseType> {
-        const copy = this.convertDateFromClient(modelingExercise);
+        const copy = ModelingExercise.convertDateFromClient(modelingExercise);
         return this.http
```

Comment:
in the other places you invoke `Exercise.convert...`. In this case, you are using `ModelingExercise.convert...`
I think this should be consistent `Exercise.convert...`